packages/web/server/queries/numia/inflation.ts

```diff
@@ -0,0 +1,23 @@
+import { apiClient } from "@osmosis-labs/utils";
+
+const NUMIA_URL = "https://public-osmosis-api.numia.xyz/apr";
```

Comment:
Perhaps we could move this URL to the index file so it can more easily be found? We've done that in other query folders.

Also, there may be a base URL in process.env? Then we could default to this hardcoded value if not.